app/models/shipit/pull_request.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Shipit
+  class PullRequest < Record
+    include DeferredTouch
+
+    belongs_to :stack
+    belongs_to :user
+    belongs_to :head, class_name: 'Shipit::Commit', optional: true
+
+    has_many :pull_request_assignments
```

Comment:
I don't see this used anywhere either. The data is collected but as far as I can tell nothing uses it.

Reply:
I mean it's exposed in the API, but that's about it.

Reply:
Yup, it's intended to expose the collection of pull request assignees through Hook emission to external systems. In our case, we have a system that receives shipit-engine PullRequest hooks and sends notifications to the PullRequest owner and assignees about the "readiness" of a Review Stack for reviews. In this case we use the `PullRequest#assignees` collection to determine part of the recipient list for notifications. 

Does that help add some color?

Reply:
This is used by consumers of Shipit hooks in order to send notifications about the deployment of a review stack to the assignees of a PR. It would be very inconvenient to have to store this separately and emit custom hooks from the wrapper application to achieve this.

Reply:
Ok, that's what I suspected. I'm a bit on the fence about this, mostly because it requires 2 extra tables. If it was an inline attribute it wouldn't mind a much.

Is having to query the GitHub API to get the extra details really a big deal? I think we do that quite a bit in our hook consumers. Or alternatively do you think it could be done as a private extension?

Reply:
> Is having to query the GitHub API to get the extra details really a big deal?

Our hook consumers (via HTTP) don't have access to the Github API.